src/ViewInstance.js

```diff
@@ -450,6 +452,8 @@ class ViewInstance extends React.Component {
           .then(onSuccess)
           .catch(onError);
       }, 2000);
+
+      this.setState({ intervalId: interval });
```

Comment:
There is no need to use state, we don't want to change anything on the UI, so let's use `this.intanceSharingInterval = setInterval(...)`

Reply:
what about now?